docs/plugins/sycl.rst

```diff
@@ -0,0 +1,86 @@
+SYCL plugin
+===========
+
+The SYCL plugin allows building interfaces that implement algorithms in SYCL.
+
+Build
+-----
+
+During the CMake configuration the flag ``CMAKE_BUILD_PLUGIN_SYCL=on`` needs to be set. CMake will check whether our compiler is SYCL compatible and will look for available SYCL target architectures. By default, SPIR 64-bits targets are looked for, but this can be configured with the ``SYCL_POSSIBLE_TARGETS`` variable. For example, to target CUDA backends, we should set this variable to ``nvptx64-nvidia-cuda-sycldevice``.
```

Comment:
```suggestion
During the CMake configuration the flag ``ACTS_BUILD_PLUGIN_SYCL=on`` needs to be set. CMake will check whether our compiler is SYCL compatible and will look for available SYCL target architectures. By default, SPIR 64-bits targets are looked for, but this can be configured with the ``SYCL_POSSIBLE_TARGETS`` variable. For example, to target CUDA backends, we should set this variable to ``nvptx64-nvidia-cuda-sycldevice``.
```
I guess

Reply:
yep